substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -1008,6 +1049,29 @@ def helloworld(args, config=None):
     )
 
 
+@mx.command(suite_name=suite.name, command_name='debuginfotest', usage_msg='[options]')
+def debuginfotest(args, config=None):
+    """
+    builds a debuginfo Hello native image and tests it with gdb.
+    """
+    parser = ArgumentParser(prog='mx debuginfotest')
+    all_args = ['--output-path', '--javac-command', '--build-only']
+    masked_args = [_mask(arg, all_args) for arg in args]
+    parser.add_argument(all_args[0], metavar='<output-path>', nargs=1, help='Path of the generated image', default=[svmbuild_dir(suite)])
+    parser.add_argument(all_args[1], metavar='<javac-command>', help='A javac command to be used', default=mx.get_jdk().javac)
```

Comment:
Providing a custom `javac-command` does not make much sense. Since your test is in the `com.oracle.svm.test` project it will be built automatically as part of running `mx build` anyway (building it again is redundant). You can use `classpath('com.oracle.svm.test')` to get the classpath argument you need to pass to native-image when you want to build `Hello`.